windowing.py

```diff
@@ -205,3 +205,84 @@ def cropped_np(np_sample, start, end):
     end_index = np.where(nanos == end)
     cropped = [np_sample[:,  start_index[0][0]:end_index[0][0] + 1]]
     return cropped
+
+
+def place_data(np_sample, data_key, data_label, dictionary, positive, window_size, stride):
+    """
+        Description: windows the np_sample (np array) with desired specifications and then
+                     places the windows into dictionary["data_key"] and creates a
+                     corresponding number of labels to dictionary["data_label"]
+
+        Args:
+            np_sample -- np array
+            data_key -- String (key within dictionary to place windows: dictionary[data_key])
+            data_label -- String (key within dictionato place label: dictionary[data_label])
+            dictionary -- dict (dictionary where windows and labels will be added to)
+            positive -- boolean (binary label of windows)
+            window_size -- int
+            stride -- int (# of samples to jump before creating next window)
+
+
+        Return:
+            dictionary -- dict (updated dictionary with added windows and labels)
+        """
+    rows = len(np_sample)
+    windowed_data = view_as_windows(np_sample, (rows, window_size), stride)
+    dictionary[data_key].extend(windowed_data[0])
+    if (positive):
+        pos_or_neg = np.ones(len(windowed_data[0]))
+    else:
+        pos_or_neg = np.zeros(len(windowed_data[0]))
+    dictionary[data_label].extend(pos_or_neg)
+    return dictionary
+
+
+def choose_dataset(data_count, num_test, num_train, num_validation):
+    """
+        Description: determines where the current sample shoud be placed (training, testing, 
+                                  validation)
+
+        Args:
+            data_count -- int (should always been in range 
+            				[0 -- num_test + num_train + num_validation])
+            num_test -- int (# in test set = num_test/(num_test + num_train + num_validation))
+            num_train -- int (# in test set = num_train/(num_test + num_train + num_validation))
+            num_validation -- int (# in test set = num_validation/(num_test + num_train + num_validation))
+
+        Return:
+            string ("test, "train", "validation" ) for where the current sample should be placed
+        """
+    if (data_count < num_test):
+        return "test"
+    if (data_count < num_train + num_test and data_count >= num_test):
+        return "train"
```

Comment:
So whether a dataset is in train or test depends on which section of the dataset it appears in?

I don't think is a huge deal for the current data, since that should be shuffled before Gautam exports it and thus is random anyways. But in general, unless you are specifically interested in temporal/sequence relations, it's good practice to randomize the assignment.